example-apps/kitchen-sink-example/source/components/form-controls/control-form.ts

```diff
@@ -1,16 +1,15 @@
+import {Component} from '@angular/core';
 
-import { Component } from '@angular/core';
-
-import {FORM_PROVIDERS, REACTIVE_FORM_DIRECTIVES, FormControl, FormGroup}
- from '@angular/forms';
+import {
+  FormControl,
+  FormGroup,
+} from '@angular/forms';
 
 @Component({
   selector: 'control-form',
-  directives: [REACTIVE_FORM_DIRECTIVES],
-  providers: [FORM_PROVIDERS],
   template: `
     <div>
-        <form (submit)="onSubmit(formControl.value)"
+        <form (submit)="onSubmit(formGroup.value)"
```

Comment:
As a note to reviewer, I think this just fixes an existing bug. There was no `formControl` property on the component before I started, and it works now that it's `formGroup`.